Hangman/Hangman/main.swift

```diff
@@ -8,5 +8,89 @@
 
 import Foundation
 
-print("Hello, World!")
+
+var chances = 6
+let allTheWords = ["apple"]
+let alphabet: Array<String> = ["a","b", "c", "d","e", "f", "g" ,"h", "i" , "j", "k","l" ,"m", "n", "o", "p", "q", "r", "s", "t", "u" , "v" , "w", "x", "y", "z" ]
+var randomWord = allTheWords.randomElement() ?? "able"
+var hiddenWord: Array<Character> = Array(repeating: "_", count: randomWord.count )
+var gameState = true
+var emptyArr = [Character]()
+
+
+func resetExit() {
+print("Do you want to play again? y/n")
+let yesOrNo = readLine()?.lowercased() ?? ""
+       switch yesOrNo {
+       case "y":
+           print("you selected yes")
+           randomWord.removeAll()
+           randomWord = allTheWords.randomElement() ?? "able"
+           hiddenWord.removeAll()
+           hiddenWord = Array(repeating: "_", count: randomWord.count )
+           chances = 6
+           emptyArr.removeAll()
+       case "n":
+           print("Goodbye")
+           exit(20)
+       default:
+           print("Please enter a y/n")
+           resetExit()
+       }
+}
+
+print("Welcome to Hangman")
+sleep(1)
+
+gameLoop: while gameState == true && chances > 0{
```

Comment:
Great use of a while loop! I so wish that I could have gotten the full picture of playing your game with a full array of allTheWords